lib/cmd/fh3/preview/show.js

```diff
@@ -0,0 +1,36 @@
+/* globals i18n */
+var util = require('util');
+var fhreq = require("../../../utils/request");
+var fhc = require("../../../fhc");
+var exec = require("../../../utils/exec.js");
+
+module.exports = {
+  'desc' : i18n._('Show the App Preview of a client app'),
+  'examples' :
+    [{
+      cmd : 'fhc preview show --app=<app>',
+      desc : "Show the preview of the <app>"
+    }],
+  'demand' : ['app'],
+  'alias' : {
+    'app': 'a',
+    0: 'app'
+  },
+  'describe' : {
+    'app' : i18n._("Unique 24 character GUID of the application")
+  },
+  'customCmd' : function(params,cb) {
+    var url = fhreq.getFeedHenryUrl() + 'box/srv/1.1/wid/' + fhc.curTarget + '/studio/' + params.app + '/container';
+    // TODO: This is almost the only place this gets used - deprecate.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS1135)